src/components/ui/InteractWithBurner.tsx

```diff
@@ -0,0 +1,176 @@
+import { AccentText, StatusText } from 'components/ui/Text'
+import { displayFrom, displayTo } from 'helpers/visibilityClassnames'
+import { useState } from 'preact/hooks'
+import Arrow from 'icons/Arrow'
+import Button from 'components/ui/Button'
+import ButtonProps from 'models/ButtonProps'
+import InfoSeal from 'icons/InfoSeal'
+import SmallInfoSeal from 'icons/SmallInfoSeal'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  display,
+  dropShadow,
+  flexDirection,
+  gap,
+  inset,
+  justifyContent,
+  margin,
+  maxWidth,
+  padding,
+  position,
+  textColor,
+  width,
+  zIndex,
+} from 'classnames/tailwind'
+
+const backgroundWrapper = classnames(
+  backgroundColor('bg-primary-dimmed'),
+  padding('p-6'),
+  display('flex'),
+  borderRadius('rounded-lg'),
+  maxWidth('max-w-full', 'lg:max-w-body'),
+  width('w-full', 'md:w-body'),
+  dropShadow('drop-shadow-info-card')
+)
+
+const buttonsWrapper = classnames(
+  display('flex'),
+  alignItems('items-center'),
+  margin('mt-4'),
+  gap('gap-x-4')
+)
+
+const positionWrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  width('w-full')
+)
+
+const headerTextWrapper = (show: boolean) =>
+  classnames(
+    display('flex'),
+    gap('gap-x-1'),
+    justifyContent('justify-between'),
+    alignItems(show ? 'items-center' : 'items-start')
+  )
+
+const contentWrapper = classnames(
+  position('fixed'),
+  inset('left-0', 'lg:left-9', 'bottom-9'),
+  zIndex('z-50'),
+  padding('p-3')
+)
+
+const infoSealWrapper = classnames(
+  displayFrom('md'),
+  margin('mr-4'),
+  dropShadow('drop-shadow-info-seal')
+)
+
+const ActionsButtons = ({
+  tertiaryButton,
+  primaryButton,
+}: {
+  tertiaryButton?: ButtonProps
+  primaryButton?: ButtonProps
+}) => {
+  return (
+    <div className={buttonsWrapper}>
+      {primaryButton?.text && (
+        <Button small type="primary" onClick={primaryButton.action}>
+          {primaryButton.text}
+        </Button>
+      )}
+      <Button
+        type="tertiary"
+        gradientFont
+        small
+        onClick={tertiaryButton?.action}
+      >
+        {tertiaryButton?.text}
```

Comment:
If you don't pass the text, the button and the container will be there, taking it's place, is this okay?

Reply:
fixed, ty